core/src/main/java/feign/Param.java

```diff
@@ -38,6 +38,12 @@
    */
   Class<? extends Expander> expander() default ToStringExpander.class;
 
+  /**
+   * Specifies whether argument is already encoded
+   * Value be ignored for headers (headers are never encoded)
```

Comment:
probably should also put `@see QueryMap#encoded` as well putting a see link there back to here